pyproject.toml

```diff
@@ -34,6 +34,7 @@ readme = {file = ["README.md"]}
 tests = [
     "pytest",
     "pytest-cov",
+    "hera-calibration",
```

Comment:
Are you sure you want to do this? Is a circular dependency a concern?

Reply:
This seems to be fine -- `hera-calibration` is only installed for the `tests` extra, and I tested this on my system and everything works fine. If we don't list it here we need to manually install it when running tests.